_posts/en/newsletters/2020-12-09-newsletter.md

```diff
@@ -120,12 +120,56 @@ just be more laid back.-->{% endcomment %}
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jonatack
+[Periodically make block-relay connections and sync headers][review club
+#19858] is a PR ([#19858][Bitcoin Core #19858]) by Suhas Daftuar to make eclipse
+attacks more difficult by increasing the existing outbound peer rotation beyond
+the current logic of stale tip detection and replacement with a new full-relay
+connection. The proposal initiates outbound connections every 5 minutes on
+average to sync headers and potentially learn of new blocks, in which case it
+rotates out an existing block-relay-only peer in favor of the new peer. This
+would raise the cost to an attacker to sustain a partitioning attack against a
+node, because as long as the node has at least one honest peer in its addrman,
```

Comment:
I think it would be nice for `addrman` to have a link. Perusing the first pages of search results, I dont personally have a suggestion for such a link, but perhaps someone else does. If not, ignore me!